crates/bevy_reflect/src/lib.rs

```diff
@@ -517,6 +518,9 @@ pub mod prelude {
         ReflectDeserialize, ReflectFromReflect, ReflectPath, ReflectSerialize, Struct, TupleStruct,
         TypePath,
     };
+
+    #[cfg(feature = "functions")]
+    pub use crate::func::IntoFunction;
```

Comment:
While I was at it, I also included a prelude export for `IntoFunction` because I feel it would be useful to not have to import it every time. If we don't want to include it in the prelude, though, let me know and I can revert this!